features/users.feature

```diff
@@ -2,6 +2,17 @@ Feature: Users
 
   @javascript
   Scenario: Logged-in non-developer user visits account page
```

Comment:
The `Logged-in non-developer user visits account page` scenario does not need to change for this story. The scenario that should be updated is `Registering as a new affiliate user without government affiliated email address`, which tests the behavior for a user with a non-.gov/.mil email address, who is not yet approved.